<ui version="4.0" >
 <class>CatManDlgClass</class>
 <widget class="QDialog" name="CatManDlgClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CatManDlg</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Category Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cmd_le_cat_name" >
       <property name="text" >
        <string>newcat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Default Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="cmd_le_default_directory" >
         <property name="text" >
          <string>C:\NGDownloads\ OR ~/.nullget/NGDownload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmd_tb_change_director" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Select the category in which the new category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="cmd_tv_cat_tree" >
       <property name="contextMenuPolicy" >
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CatManDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>367</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel" >
     <x>317</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CatManDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>367</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel" >
     <x>343</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
